src/Univ2LpOracle.sol

```diff
@@ -169,8 +169,12 @@ contract UNIV2LPOracle {
         src  = _src;
         zzz  = 0;
         wat  = _wat;
-        dec0 = uint8(ERC20Like(UniswapV2PairLike(_src).token0()).decimals());  // Get decimals of token0
-        dec1 = uint8(ERC20Like(UniswapV2PairLike(_src).token1()).decimals());  // Get decimals of token1
+        uint256 _normalizer0 = 10 ** sub(18, uint8(ERC20Like(UniswapV2PairLike(_src).token0()).decimals()));  // Calculate normalization factor of token0
+        require(_normalizer0 >= 1);
```

Comment:
Use a revert message here: "UNIV2LPOracle/invalid-oracle-0-decimals"

Reply:
Actually this `require` is straight-up unnecessary, since `0 ** 0 == 1`